service/application/src/main/java/com/solace/maas/ep/event/management/agent/config/eventPortal/EventPortalProperties.java

```diff
@@ -5,16 +5,19 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.PropertySource;
 
+import java.util.List;
```

Comment:
I don't think you need the changes in this file. The ConfigurationProperties are applied after the ConditionalOnProperty, so it won't make a difference.